src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/ndelius/Sentence.kt

```diff
@@ -1,21 +1,18 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.ndelius
 
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Term as IntegrationApiTerm
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.SentenceLength as IntegrationApiLength
 
 data class Sentence(
   val date: String? = null,
   val description: String? = null,
   val length: Int? = null,
   val lengthUnits: String? = null,
 ) {
-  fun toTerm(): List<IntegrationApiTerm> {
-    return when (this.lengthUnits?.lowercase()) {
-      "years" -> listOf(IntegrationApiTerm(years = this.length))
-      "months" -> listOf(IntegrationApiTerm(months = this.length))
-      "weeks" -> listOf(IntegrationApiTerm(weeks = this.length))
-      "days" -> listOf(IntegrationApiTerm(days = this.length))
-      "hours" -> listOf(IntegrationApiTerm(hours = this.length))
-      else -> listOf(IntegrationApiTerm())
-    }
+  fun toLength(): IntegrationApiLength {
+    return IntegrationApiLength(
+      duration = this.length,
+      units = this.lengthUnits?.lowercase(),
```

Comment:
I'm wondering whether we should keep it as uppercase. In our images endpoint for enum values like view and orientation they are uppercase. Wondering if our convention should be if it's an enum value, it should be uppercase?